front/src/js/message-list.js

```diff
@@ -3,15 +3,27 @@ const messages = {
     props: ['messages'],
     methods: {
         humanDate(ts,now){
+            console.log(ts);
```

Comment:
Remove this log please.

Reply:
we must now keep the "now" argument? because it is not used